src/server/verification/api/ZoomAPI.js

```diff
@@ -126,33 +115,56 @@ class ZoomAPI {
         return encodeURIComponent(parameterValue)
       }
 
+      log.debug('Calling Zoom API:', config)
+
       return {
         ...config,
         params: searchParams,
-        url: (url || '').replace(/:(\w[\w\d]*?)/g, substituteParameter)
+        url: (url || '').replace(/:(\w[\w\d]+)/g, substituteParameter)
       }
     })
   }
 
   _configureResponses() {
     const { response } = this.http.interceptors
-    const responseInterceptor = ({ data }) => merge(...Object.values(pick(data, 'meta', 'data')))
 
-    response.use(responseInterceptor, async exception => {
-      const { response, message } = exception
+    response.use(response => this._responseInterceptor(response), exception => this._exceptionInterceptor(exception))
+  }
 
-      if (response && isPlainObject(response.data)) {
-        const zoomResponse = responseInterceptor(response)
-        const { message: zoomMessage } = zoomResponse
+  _responseTransformer(response) {
+    const { data } = response
 
-        exception.message = zoomMessage || message
-        exception.response = zoomResponse
+    return merge(...Object.values(pick(data, 'meta', 'data')))
+  }
+
+  _responseInterceptor(response) {
+    log.debug('Received response from Zoom API:', response)
```

Comment:
we need some kind of context for these logs
in general we need context for all places where you log
1. either pass down the log object from express "req" this way each log call that happens in same API call will have same unique id
2. add user identifier